NYPLAudiobookToolkit/UI/AudiobookPlayerViewController.swift

```diff
@@ -459,6 +354,221 @@ let SkipTimeInterval: Double = 15
         buttonItem.accessibilityTraits = UIAccessibilityTraits.button
         return buttonItem
     }
+  
+    func setupUI() {
```

Comment:
@ettore This and the functions below are from `viewDidLoad` method, I just refactored them into smaller pieces and added a bookmark button to the nav bar.

Reply:
I have only make minor UI changes to it so nothing in particular